pegasus/sites.v3/code.org/public/yourschool.haml

```diff
@@ -52,6 +55,9 @@ social:
 
 #partners
   %hr
+
+  %p= @hoc_data
```

Comment:
Is this replacing something else? I'm not quite sure I'm following how the yourschool file comes into play

Reply:
Ah sorry, I still had one more commit to push where I remove these changes (this was for showing the output in a non-hoc site location)